### PR TITLE
Iteration IMU Transformations AL Article

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: alpha-lab/imu-transformations/index.md [hunk 1]
```diff
@@ -57,11 +53,11 @@ def transform_imu_to_world(imu_coordinates, imu_quaternions):
         ])
 ```
 
-Now that we have the `transform_imu_to_world` function, let's use it!
+### Example: Heading Vectors in World Coordinates
```
Comment (marc-tonsen) [inline alpha-lab/imu-transformations/index.md:56]: I changed all the applications of the methods we defined to examples with a ### header. This way in the ## sections we introduce the transformation function and in the ### sections we show how to use it.

File: alpha-lab/imu-transformations/index.md [hunk 2]
```diff
@@ -85,17 +83,16 @@ accelerations_in_world = transform_imu_to_world(
 
 ## Scene to World Coordinates
 
-Neon simultaneously records data in the scene camera and IMU coordinate systems, making it possible to study the relationship between head and eye movements.
+A lot of the data generated by Neon is provided in the scene camera's coordinate system, including e.g. gaze, fixation, and eye state data. This coordinate system is **not** equal to the IMU's coordinate system! There is a translation between them (simply because there is a physical distance between the camera and the IMU in the module) and also a rotation (because of how the scene camera's coordinate system is defined).
 
-To facilitate the comparison, it can be useful to represent these data streams in the same coordinate system. An important step is accounting for [the fixed 102 degree rotation offset between the scene camera and IMU coordinate systems](https://docs.pupil-labs.com/neon/data-collection/data-streams/#movement-imu-data), as depicted below.
+The rotation is a 102 degree rotation around the x-axis of the IMU coordinate system and the translation is along the vector `(0.0 mm, -1.3 mm, -6.62 mm)`.
```
Comment (marc-tonsen) [inline alpha-lab/imu-transformations/index.md:88]: IMO the case including the translation is the default case and the application to 3D directions which should skip the translation is the special case. Introducing it like this makes the concept of "scene cam coords are not the same as IMU coords" easier to understand I think.

File: alpha-lab/imu-transformations/index.md [hunk 3]
```diff
@@ -202,93 +196,17 @@ def spherical_to_cartesian_scene(elevations, azimuths):
...
-def gaze_2d_to_world(gaze_points_2d, scene_camera_matrix, scene_distortion_coefficients, imu_quaternions):
-    cart_gazes_in_scene = undistort_and_unproject(
-        gaze_points_2d, scene_camera_matrix, scene_distortion_coefficients
-    )
-    return transform_scene_to_world(
-        cart_gazes_in_scene, imu_quaternions
-    )
-```
-
-## World Spherical Coordinates
```
Comment (marc-tonsen) [inline alpha-lab/imu-transformations/index.md:249]: This too is not related to IMU transformations. I'm not sure we'd want to cover the transformation between cartesian and spherical coordinates anywhere explicitly in our docs, as this is an algebra problem that's covered a lot on the internet and it is not required that often in eye tracking. Happy to discuss this though!

Comment (rennis250) [reply alpha-lab/imu-transformations/index.md:249]: @marc-tonsen In our case, I say we need it, because Neon's spherical coordinate conventions break with traditional spherical coordinate conventions in subtle ways. Considering the IMU seems to be a frequent source of confusion for users, I would vote to make things as easy as possible for them. It's also been a request from two users so far.

File: alpha-lab/imu-transformations/index.md [hunk 4]
```diff
@@ -202,93 +196,17 @@ def spherical_to_cartesian_scene(elevations, azimuths):
...
-Now we have the tools to convert 3D gaze data to world coordinates:
+Now we can transform the data to world coordinates. Since we are dealing with 3D directions, rather than 3D points here, it does not make sense to apply the translation that we used in the `transform_scene_to_world` function above. We are thus setting it to zero here.
 
 ```python
 def gaze_3d_to_world(gaze_elevation, gaze_azimuth, imu_quaternions):
     cart_gazes_in_scene = spherical_to_cartesian_scene(gaze_elevation, gaze_azimuth)
-    return transform_scene_to_world(cart_gazes_in_scene, imu_quaternions)
-```
-
-## 2D Gaze to World Coordinates
```
[the remark this answers is not in the document]
Comment (rennis250) [reply alpha-lab/imu-transformations/index.md:213]: @marc-tonsen I vote for removing 2D Gaze to World Coordinates, as we already perform the 2D to 3D gaze transformation automatically in Cloud, Neon Player, and pl-rec-export.

File: alpha-lab/imu-transformations/index.md (hunk 1 above)
Comment (rennis250) [reply alpha-lab/imu-transformations/index.md:56]: @marc-tonsen Thanks!

File: alpha-lab/imu-transformations/index.md (hunk 2 above)
Comment (rennis250) [reply alpha-lab/imu-transformations/index.md:88]: @marc-tonsen Agreed!
